index.js

```diff
@@ -33,6 +37,8 @@ if (Platform.OS === 'web') {
         VizbeePlayerState,
         VizbeeTrackStatus,
         VizbeeTrackInfo,
+        VizbeeSignInDelegate,
+        VizbeeSignInInfo,
```

Comment:
Same as above, let's add these after the VizbeeCastButton

Reply:
Re-organized the exports as per flow